### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by amymok**
Make sure readme is detail enough for new team to pick up the project again

## Additions

- Add workflow to start elastic search

## Review

- @virginiacc

## Checklist

* [X] Changes are limited to a single goal (no scope creep)
* [X] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices 
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +183,19 @@ $ npm install
 $ grunt
 ```
 
+### Elastic Search workflow
+Elastic Search needs to be running to display wordpress content:
+1. Use the sheer virtualenv:
+    ```bash
```
Comment (virginiacc) [inline README.md]: When I look at the readme in the browser, step 1 appears on the same line as the previous sentence. Does there need to be a new line between them?

**Comment by virginiacc** [comment]
Looks good! :+1: 

I had one formatting suggestion, and then this should be ready to merge...

**Comment by amymok** [comment]
@virginiacc Thanks for the catch.  This should be good now!

**Comment by virginiacc** [comment]
Looks great!

**Pull request merged**